v3/bind.go

```diff
@@ -576,7 +576,7 @@ type GSSAPIClient interface {
 	// reply token is received from the server, passing the reply token
 	// to InitSecContext via the token parameters.
 	// See RFC 4752 section 3.1.
-	InitSecContext(target string, token []byte) (outputToken []byte, needContinue bool, err error)
+	InitSecContext(target string, token []byte, APOptions []int) (outputToken []byte, needContinue bool, err error)
```

Comment:
As mentioned in the comment on the implementation; let's not modify (break) an existing API - we should add a _new_ method if we really need to.